src/cloud-api-adaptor/libvirt/config_libvirt.sh

```diff
@@ -12,6 +12,9 @@ set -o pipefail
 source /etc/os-release || source /usr/lib/os-release
 ARCH=$(uname -m)
 TARGET_ARCH=${ARCH/x86_64/amd64}
```

Comment:
I'm wondering if it would be neater to use the same style for x86 as well. Maybe something like:
```
if [[ "${ARCH}" == "x86_64" ]]; then
    TARGET_ARCH=amd64
elif [[ "${ARCH}" == "aarch64" ]]; then
    TARGET_ARCH=arm64
else 
  TARGET_ARCH=${ARCH}
fi
```
?